test/ptp/common.go

```diff
@@ -41,3 +42,27 @@ func getDevInfo(api *client.ClientSet, intf string, ptpPod *corev1.Pod) (vendor,
 	logrus.Infof("vendor: %s, device ID: %s, ttyGNSS: %s", vendor, device, ttyGNSS)
 	return
 }
+
+func getOriginUrl() (origin_url string) {
```

Comment:
Add some log debug messages for each error case here, if only to make the error visible. As it is errors are silently discarded if they are not `nil`.

Reply:
Added Ginkgo prints for critical link component errors, just like the link this will only work with "ginkgo -v" verbose logging.